doc/ci-cd/deployment.md

```diff
@@ -103,31 +118,64 @@ If not urgent (but still can't wait until the next MEP), it's better for the use
+- List all tags of the version `X.Y.Z`, tags of type: `vX.Y.Z-W` (`git fetch --tag`and then `git tag | grep vX.Y`)
 - Two cases:
-  - There is already a previous hotfix on this version (`hotfix-production-vX.X.X.Y`): checkout on the tag with the biggest `Y`
-  - It is the first hotfix on this version : Checkout on the tag `vX.X.X` (could be `patch/vX.X.X`) with the bigger minor `X`
+  - There is already a previous hotfix on this version (`hotfix-production-vX.Y.Z.W`): checkout on the tag with the biggest `W`
+  - It is the first hotfix on this version : Checkout on the tag `vX.Y.Z` (could be `patch/vX.Y.Z`) with the biggest minor `Y`
 - Cherry-pick all the commits of the feature `git cherry-pick <commit-hash>`
 - Check if tests are OK `yarn test`
-- `git tag hotfix-staging-vX.X.X-(Y+1)`
-- `git push origin hotfix-staging-vX.X.X-(Y+1) --no-verify`: this will deploy it to `staging`
-- Validate the fix with the PO on staging app (version X.X.X)
+- `git tag hotfix-staging-vX.Y.Z-(W+1)`
+- `git push origin hotfix-staging-vX.Y.Z-(W+1) --no-verify`: this will deploy it to `staging`
+- Validate the fix with the PO on staging app (version X.Y.Z)
 - If it is OK for the PO, deploy it to production:
-- `git tag hotfix-production-vX.X.X-(Y+1)`
-- `git push origin hotfix-production-vX.X.X-(Y+1) --no-verify`: this will deploy it to `production`
-- ⚠️ Check your code push targets the actual production version (one code push targets only one version)
+  - `git tag hotfix-production-vX.Y.Z-(W+1)`
+  - `git push origin hotfix-production-vX.Y.Z-(W+1) --no-verify`: this will deploy it to `production`
+  - ⚠️ Check your code push targets the actual production version (one code push targets only one version)
 
 ### Troubleshooting
 
 <details>
-  <summary>I don't see my CodePush on staging/prod app</summary>
+ <summary>I don't see my CodePush on staging/prod app</summary>
 
 Check if you can find it on AppCenter. Example for [staging iOS][8].
 
 ![img](./CodePushOnAppCenter.png)
 
 </details>
 
+## Sprints, MeS and MeP
+
+Iteration 300.
+
+- tag is 300
+- Testing is under version number 301
+- Staging is under version number 300
+- Prod is under version number 300
+
+We run processes for 301 (MeS).
+
+- Set Tag under version « 301 »
+- Testing is under version number 302, compilation and merges on master prepare the 302's future version well
+- Staging is compiled from tag's 301, we are testing version 301
+- Prod is under version number 300
+- New iteration goes on 302
+
+We run processes for 302 (MeS and MeP).
+
+- Set Tag under version « 302 »
+- Testing is under version number 303, compilation and merges on master prepare the 303's future version well
+- Staging is compiled from tag's 302, we are testing version 302
+- Prod is compiled from tag's 302, we publish result of 301+302
+- New iteration goes on 303
+
+Nous lançons la mes de la version 303 (MeS).
```

Comment:
the rest of le document is en anglais

why ici is en french ?

Reply:
But oui you are raison :)